README.md

```diff
@@ -20,6 +20,16 @@ Thank you for your interest in contributing to this project!
 
 Please review the [contribution guidelines](docs/contributing.md), and note that all contributions must adhere to the [code of conduct](docs/code-of-conduct.md).
 
+### Testing changes locally
```

Comment:
Do you want to move this into this [contributing process step](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/03188791e1548defe894f43aadb81f18df7120d4/docs/contributing.md)? So we have contributing guide together?

Reply:
Or it's actually better if we move that all content from contributing page to the main page [README.md](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/README.md)?

Reply:
Good idea. I'll move this to the contributing doc.